File: app/assets/javascripts/backbone/models/Flight.js
var app = app || {};

app.Flight = Backbone.Model.extend({
	urlRoot: '/flights',
	defaults: {
		flight_number: 'Unassigned Flight Number',
		date: '1st January 2000',
		time: '00:00',
		origin: 'Unassigned Origin',
		destination: 'Unassigned Destination'
	}
});

File: app/models/flight.rb
# == Schema Information
#
# Table name: flights
#
#  id            :integer          not null, primary key
#  flight_number :string
#  origin        :string
#  destination   :string
#  date          :date
#  time          :time
#  plane_id      :integer
#  created_at    :datetime         not null
#  updated_at    :datetime         not null
#

class Flight < ActiveRecord::Base
	belongs_to :plane
	has_many :reservations
end
